File: app/src/main/java/mishka/mingeo/data/model/Pumping.java
package mishka.mingeo.data.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.arch.persistence.room.TypeConverters;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;


@Entity
public class Pumping implements Serializable, DatabaseEntity{
    @PrimaryKey(autoGenerate = true)
    public int id;

    public int getId() {
        return id;
    }
}

File: app/src/main/java/mishka/mingeo/data/model/Pumping.kt
package mishka.mingeo.data.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.arch.persistence.room.TypeConverters
import org.joda.time.Instant
import java.io.Serializable

@Entity
@TypeConverters(DateConverter::class)
class Pumping: Serializable {
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0

    var pumpPower: Float = 0f

    var centralBoreholeId: Long? = null

    var startPumpingTime: Instant? = null

}
